src/sentry/rules/actions/sentry_apps/notify_event.py

```diff
@@ -62,6 +62,23 @@ def _get_setting_value(self, field_name: str) -> str | None:
             None,
         )
 
+    def _get_sentry_app_installation_uuid(self) -> Any:
+        sentry_app_installation_uuid = self.data.get("sentryAppInstallationUuid")
+        if not sentry_app_installation_uuid:
+            raise ValidationError("Missing attribute 'sentryAppInstallationUuid'")
+        return sentry_app_installation_uuid
+
+    def _get_alert_rule_component(self, sentry_app_id: int, sentry_app_name: str) -> Any:
```

Comment:
Getting alert rule component was moved to a function to avoid redundant code (used in existing `self_validate` and new `render_label` functions